docs/apis-tools/web-modeler-api/index.md

```diff
@@ -92,16 +84,23 @@ To authenticate for the API, generate a JWT token and pass it in each request; g
    }
    ```
 
-### Limitations of Beta Version
+## Limitations
 
-When using Web Modeler API beta:
+When using Web Modeler API:
 
-- You will not receive a warning when deleting a file, a folder, or a project. This is important, because deletion cannot be undone.
+- You will not receive a warning when deleting a file, a folder, or a project.
+  This is important, because deletion cannot be undone.
 - You will not receive a warning about breaking call activity links or business rule task links when moving files or folders to another project.
   Breaking these links is considered harmless. The broken links can be manually removed or restored in Web Modeler. This operation is also
   reversible - simply move the files or folders back to their original location.
-- You will not immediately observe a new project you created via the API. This is because the project has no collaborators. To remedy this, the
-  org owner can activate [super-user mode](../../components/modeler/web-modeler/collaboration/#super-user-mode) and assign collaborators.
+
+## Rate Limiting
+
+In the cloud, the Web Modeler API uses rate limiting to control traffic.
+The limit is 240 requests per minute.
+Surpassing this limit will result into a `HTTP 429 Too Many Requests` response.
+
+On Self-Managed instances no limits are enforced.
```

Comment:
Is this configurable in Self-Managed? Or would this depend on their Self-Manage installation and architecture?

Reply:
It's not configurable, depends on the individual infrastructure.